message.go

```diff
@@ -251,8 +251,8 @@ type MessageSend struct {
 // is also where you should get the instance from.
 type MessageEdit struct {
 	Content         *string                 `json:"content,omitempty"`
-	Components      []MessageComponent      `json:"components"`
-	Embeds          []*MessageEmbed         `json:"embeds"`
+	Components      []*MessageComponent     `json:"components,omitempty"`
```

Comment:
Same as `MessageSend.Components`. However, since the field could be `[]` the array should be a pointer instead.
```suggestion
	Components      *[]MessageComponent     `json:"components,omitempty"`
```